src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Shooter;
+
+
+public class ShootCommand extends Command {
+
+    private final Shooter shooter;
+    private final XboxController controller;
+
+    private double leftTrigger;
+
+    public ShootCommand(Shooter shooter, XboxController controller) {
+        // each subsystem used by the command must be passed into the
+        // addRequirements() method (which takes a vararg of Subsystem)
+
+        this.shooter = new Shooter();
```

Comment:
We dont need to create a new instance of the shooter, we can just use the already made shooter class.